src/core/server/http/http_config.ts

```diff
@@ -91,6 +91,7 @@ export const config = {
           )
         ),
       }),
+      uuid: schema.maybe(schema.string()),
```

Comment:
I don't like having to add this here, as the `server` config namespace is now more like only the `http` config, However the current property is `server.uuid`, so I don't really see another way.

We could deprecates/rename that once the config deprecation PR lands if we want to.

Reply:
Can you add a note to the config refactoring issue just so we don't forget?

Reply:
In the legacy config, we validated this as a valid guid. Can we add that validation here as well? Should be able to specify a custom `validate` option to `schema.string()`

Reply:
Oh good catch, totally forgot about that